src/transformers/models/dpr/modeling_tf_dpr.py

```diff
@@ -743,7 +768,6 @@ def call(
         self,
         input_ids=None,
         attention_mask: Optional[tf.Tensor] = None,
-        token_type_ids: Optional[tf.Tensor] = None,
```

Comment:
Is it a previous mistake fixed here? It doesn't seem relevant to serving.

Reply:
It is a small fix yes!